dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/common/util/ServiceTestUtil.java

```diff
@@ -72,8 +72,8 @@ private static boolean isPrimitiveType(TypeDefinition td) {
                 type.equals("java.lang.Object");
     }
 
-    private static List generateParameterTypes(String[] parameterTypes, ServiceDefinition serviceDefinition) {
-        List parameters = new ArrayList();
+    private static List<Object> generateParameterTypes(String[] parameterTypes, ServiceDefinition serviceDefinition) {
```

Comment:
Because the values of the returned types are not identical and using generic type in collection is considered a good practice

Reply:
I don't know where is `List<Object>` is better than `List`.

Reply:
It will suppress the compile warning if I must give a reason, and it is considered a better practice in new code, see more discussion in [this thread](https://stackoverflow.com/a/2770692/5200161).

Reply:
Good job, Thx.